src/chesster.js

```diff
@@ -24,118 +16,16 @@ var chesster = new slack.Bot({
     config_file: config_file
 });
 
-// A helper for a very common pattern
-function directRequiresLeague(patterns, callback) {
-    chesster.hears(
-        {
-            middleware: [slack.requiresLeague],
-            patterns: patterns,
-            messageTypes: ['direct_message', 'direct_mention']
-        },
-        callback
-    );
-}
-
-/* league information */
-directRequiresLeague(
-    ['captain guidelines'],
-    leagueInfo.directResponse('formatCaptainGuidelinesResponse')
-);
-directRequiresLeague(
-    ['captains', 'captain list'],
-    leagueInfo.dmResponse('formatCaptainsResponse')
-);
-directRequiresLeague(
-    ["faq"],
-    leagueInfo.directResponse('formatFAQResponse')
-);
-directRequiresLeague(
-    ['notify mods', 'summon mods'],
-    leagueInfo.directResponse('formatSummonModsResponse')
-);
-directRequiresLeague(
-    ['^mods$', '^moderators$'],
-    leagueInfo.directResponse('formatModsResponse')
-);
-directRequiresLeague(
-    ['pairings'],
-    leagueInfo.directResponse('formatPairingsLinkResponse')
-);
-directRequiresLeague(
-    ['rules', 'regulations'],
-    leagueInfo.directResponse('formatRulesLinkResponse')
-);
-directRequiresLeague(
-    ['standings'],
-    leagueInfo.directResponse('formatStandingsLinkResponse')
-);
-directRequiresLeague(
-    ['^welcome$', 'starter guide', 'player handbook'],
-    leagueInfo.directResponse('formatStarterGuideResponse')
-);
-
-/* availability */
-chesster.hears(
-    {
-        middleware: [slack.withLeague],
-        patterns: ['available', 'unavailable'],
-        messageTypes: ['direct_message', 'direct_mention']
-    },
-    availability.updateAvailability
-);
-
-/* alternate assignment */
-chesster.hears(
-    {
-        middleware: [slack.withLeagueByChannelName],
-        patterns: ['^assign'],
-        messageTypes: ['ambient']
-    }, 
-    availability.assignAlternate
-);
-
-/* alternate unassignment */
-chesster.hears(
-    {
-        middleware: [slack.withLeagueByChannelName],
-        patterns: ['^unassign'],
-        messageTypes: ['ambient']
-    }, 
-    availability.unassignAlternate
-);
-
-/* game nomination */
-chesster.hears(
-    {
-        middleware: [slack.requiresLeague],
-        patterns: ['nomination'],
-        messageTypes: ['direct_message']
-    },
-    nomination.nomination
-);
-
-/* rating */
-
-chesster.hears(
-    {
-        patterns: [slack.appendPlayerRegex("rating", true)],
-        messageTypes: ['direct_mention', 'direct_message']
-    },
-    playerInfo.playerRating
-);
-
-chesster.hears(
-    {
-        patterns: [
-            slack.appendPlayerRegex("pairing", true)
-        ],
-        messageTypes: ['direct_mention', 'direct_message']
-    },
-    playerInfo.playerPairings(chesster.config)
-);
+const games = require('./commands/games.js')(chesster, slack);
```

Comment:
Same goes for all of these, slack is just another module. I'd rather you just imported it in the commands modules directly.